includes/multi-currency/Analytics.php

```diff
@@ -98,6 +112,34 @@ public function register_admin_scripts() {
 		);
 	}
 
+	/**
+	 * Add the list of currencies used on the store to the wcSettings to allow it to be accessed by the front-end JS script.
+	 *
+	 * @return void
+	 */
+	public function register_customer_currencies() {
+		$currencies           = $this->multi_currency->get_all_customer_currencies();
+		$available_currencies = $this->multi_currency->get_available_currencies();
+		$currency_options     = [];
+
+		foreach ( $currencies as $currency ) {
+			if ( ! isset( $available_currencies[ $currency ] ) ) {
+				continue;
+			}
```

Comment:
What do you think about creating another method like `get_available_customer_currencies` to move this there and avoid duplicated code in the future if we need to access them? I don't want to overload `Multi_Currency` class with so many methods, but this one could be handy to avoid looping over currencies to filter out those that are not available anymore.

Reply:
Make sense, better to create that method if it's needed and avoid a second loop for now 🙂 And yes, the possibility is quite remote, I was thinking of a currency which doesn't exist anymore, like a recent issue we had with a currency changing the code.